src/presets/user_account.cairo

```diff
@@ -165,7 +165,7 @@ mod ArgentUserAccount {
             self: @ContractState,
             class_hash: felt252,
             contract_address_salt: felt252,
-            threshold: usize,
+            new_threshold: usize,
```

Comment:
Whats the reason behind this renaming?

Reply:
New cairo requires the signatures of the constructor and `__validate_deploy__` to be analogous. This is the name used in the constructor, so I've used it here as well.
```cairo
#[constructor]
fn constructor(ref self: ContractState, new_threshold: usize, signers: Array<Signer>) 
```